### ISSUE TITLE
Golem changes subtask count from 8 to 5, task ends in timeout

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#4659]
## Description

**Golem Version**: 0.20.1+dev105.gf2d43b5
https://buildbot.golem.network/buildbot/#/builders/2/builds/5169

**Golem-Messages version** (leave empty if unsure): 3.7.0

**Electron version** (if used): 0.2.1

**OS** [e.g. Windows 10 Pro]: all

**Branch** (if launched from source):

**Mainnet/Testnet**: mainnet

**Priority label is set to the lowest by default. To setup higher priority please change the label**
_P0 label is set for Severity-Critical/Effort-easy
P1 label is set  for Severity-Critical/Effort-hard
P2 label is set for Severity-Low/ Effort-easy
P3 label is set for Severity-Low/Effort-hard_

**Description of the issue**:

In task with 2 frames Golem changes subtask count from 8 to 5. When settings for task are set, it is showing 8 subtasks count, also on confirmation modal. When task is started subtask count is changed to 5. Task ends in timeout, Golem computes 2 subtasks in each frame, completing both frames. 5th subtask remains in not started state, task ends in timeout.

**Actual result**:

Even though all frames are computed task will end in timeout.

**Screenshots**:

![Screenshot from 2019-08-28 17-18-41](https://user-images.githubusercontent.com/32061891/63869361-4c226a00-c9b8-11e9-8776-4a0848062939.png)

## Steps To Reproduce
_Short description of steps to reproduce the behavior:_
e.g.
1. Launch Golem, create a task small_shark.blend, apply resolution from file, set 2 frames, 8 subtasks, 10 min for task, 5 min for subtask, bid 0.1
2. Check confirmation modal
3. Start a task
4. Check task settings in GUI and from cli
5. Wait for timeout of the task 

## Expected behavior

Golem should set correct number of subtasks

## Logs and any additional context
```
>> tasks dump 8f33c1f6-c9a2-11e9-b147-5dbf74ee525e
{
  "id": "8f33c1f6-c9a2-11e9-b147-5dbf74ee525e",
  "status": "Timeout",
  "time_remaining": 263.2146700024605,
  "subtasks_count": 5,
  "progress": 0.8,
  "cost": null,
  "fee": null,
  "estimated_cost": "41666666666666670",
  "estimated_fee": "700000000000007",
  "last_updated": 1567004164.9526718,
  "name": "small_shark100",
  "type": "Blender",
  "time_started": 1567003563.6013894,
  "compute_on": "cpu",
  "preview": "/home/ederenn/.local/share/golem/default/mainnet/ComputerRes/8f33c1f6-c9a2-11e9-b147-5dbf74ee525e/tmp/current_task_preview1.PNG",
  "options": {
    "frame_count": 2,
    "output_path": "/home/ederenn/Documents/small_shark100_2019-08-28_16-46-03",
    "format": "PNG",
    "resolution": [
      400,
      400
    ],
    "frames": "1-2",
    "compositing": false,
    "samples": 1000
  },
  "timeout": "0:10:00",
  "concent_enabled": false,
  "subtask_timeout": "0:05:00",
  "bid": 0.1,
  "resources": [
    "/home/ederenn/Desktop/Host_Share_Folder/test files/small_shark100.blend"
  ]
}
>> tasks dump 20f7d3c8-c9a7-11e9-9c97-5dbf74ee525e
{
  "id": "20f7d3c8-c9a7-11e9-9c97-5dbf74ee525e",
  "status": "Computing",
  "time_remaining": 323.1784553527832,
  "subtasks_count": 5,
  "progress": 0.2,
  "cost": null,
  "fee": null,
  "estimated_cost": "41666666666666670",
  "estimated_fee": "440440000000000",
  "subtask_timeout": "0:05:00",
  "preview": null,
  "type": "Blender",
  "time_started": 1567005526.1457083,
  "last_updated": 1567005606.9337819,
  "timeout": "0:10:00",
  "bid": 0.1,
  "options": {
    "frame_count": 2,
    "output_path": "/home/ederenn/Documents/small_shark100_2019-08-28_17-18-46",
    "format": "PNG",
    "resolution": [
      400,
      400
    ],
    "frames": "1-2",
    "compositing": false,
    "samples": 1000
  },
  "compute_on": "cpu",
  "name": "small_shark100",
  "resources": [
    "/home/ederenn/Desktop/Host_Share_Folder/test files/small_shark100.blend"
  ],
  "concent_enabled": false
}
>> tasks subtasks list 20f7d3c8-c9a7-11e9-9c97-5dbf74ee525e
┌──────────────────┬────────────────────────────────────────┬────────────┬────────────┐
│  node            │  subtask id                            │  status    │  progress  │
├──────────────────┼────────────────────────────────────────┼────────────┼────────────┤
│  FB_3661         │  30683114-c9a7-11e9-8c2e-5dbf74ee525e  │  Starting  │  0.0 %     │
│  Liesse          │  307b1c26-c9a7-11e9-a68d-5dbf74ee525e  │  Finished  │  100.0 %   │
│  tintirimintiri  │  30a064b6-c9a7-11e9-a0fb-5dbf74ee525e  │  Finished  │  100.0 %   │
│  maco-nodey-1    │  30d81b14-c9a7-11e9-92d6-5dbf74ee525e  │  Starting  │  0.0 %     │
└──────────────────┴────────────────────────────────────────┴────────────┴────────────┘
>> 
```

## Proposed Solution?
_(Optional: What could be a solution for that issue)_

**Comment by ederenn** [comment]
With other even subdivisions problem continues. E.g 2 in settings 2 frames, 16 subtasks is changed to 2 frames 10 subtasks.

**Comment by etam** [comment]
fixed in #4662
